validator/val_sigcrypt.c

```diff
@@ -718,9 +718,9 @@ dnskey_verify_rrset(struct module_env* env, struct val_env* ve,
 	}
 	verbose(VERB_ALGO, "rrset failed to verify: all signatures are bogus");
 	if(!numchecked) {
-		*reason = "signature missing";
+		*reason = "signatures bogus";
```

Comment:
I see your point but it is not 100% clear from the [rfc](https://www.rfc-editor.org/rfc/rfc8914.html#name-extended-dns-error-code-10-) if code 10 means there are no RRSIGS at all (we cover that case with `num==0` above) or no rrsigs that we are expecting. I interpreted the former, but I see that the later could also be the case. It mostly boils down to the end user looking at the EDE code, I believe it would be confusing to say rrsigs are missing when at least some rrsigs are there.
If you still think that missing rrsigs is the correct code there I have no problem changing this.

Reply:
The ede code can be the bogus code. I was thinking the error text itself is not improving, when rrsigs that match certain conditions are absent. But, if the text is more germane for many cases, that would be fine for me.

Reply:
Ah, if it is the text then I agree!